onnxruntime/core/providers/openvino/ov_interface.h

```diff
@@ -10,6 +10,7 @@
 #include <utility>
 
 #include "openvino/openvino.hpp"
+#include "openvino/runtime/intel_npu/properties.hpp"
```

Comment:
does this require a minimal OV version?

Reply:
Not the file but some properties are introduced in specific OV versions.